From 123d7ac67845d82c38b7dc98987ee5d6c62ad54d Mon Sep 17 00:00:00 2001
From: Oz Tiram <oz.tiram@gmail.com>
Date: Thu, 20 Jun 2024 23:31:20 +0200
Subject: [PATCH 1/2] org.mate.ScreenSaver->org.freedesktop.ScreenSaver

Signed-off-by: Oz Tiram <oz.tiram@gmail.com>
---
 src/gs-listener-dbus.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/gs-listener-dbus.c b/src/gs-listener-dbus.c
index 520926d..3a9b02b 100644
--- a/src/gs-listener-dbus.c
+++ b/src/gs-listener-dbus.c
@@ -51,9 +51,9 @@ static DBusHandlerResult gs_listener_message_handler    (DBusConnection  *connec
         DBusMessage     *message,
         void            *user_data);
 
-#define GS_LISTENER_SERVICE   "org.mate.ScreenSaver"
-#define GS_LISTENER_PATH      "/org/mate/ScreenSaver"
-#define GS_LISTENER_INTERFACE "org.mate.ScreenSaver"
+#define GS_LISTENER_SERVICE   "org.freedesktop.ScreenSaver"
+#define GS_LISTENER_PATH      "/org/freedesktop/ScreenSaver"
+#define GS_LISTENER_INTERFACE "org.freedesktop.ScreenSaver"
 
 /* systemd logind */
 #define SYSTEMD_LOGIND_SERVICE   "org.freedesktop.login1"
@@ -1370,7 +1370,7 @@ do_introspect (DBusConnection *connection,
 
 	/* ScreenSaver interface */
 	xml = g_string_append (xml,
-	                       "  <interface name=\"org.mate.ScreenSaver\">\n"
+	                       "  <interface name=\"org.freedesktop.ScreenSaver\">\n"
 	                       "    <method name=\"Lock\">\n"
 	                       "    </method>\n"
 	                       "    <method name=\"Unlock\">\n"
-- 
2.43.2

