diff --git a/configure.ac b/configure.ac
index 848418a..8e6087f 100644
--- a/configure.ac
+++ b/configure.ac
@@ -110,17 +110,6 @@ fi
 GTK_DOC_CHECK(1.0)
 
 ###########################
-# gtester2xunit checks    #
-###########################
-
-AC_PATH_PROG([PYTHON],[python])
-AC_MSG_CHECKING(for gtester2xunit dependencies)
-if !($PYTHON -c "import libxslt, libxml2" 2> /dev/null); then
-  AC_MSG_RESULT([no])
-  AC_MSG_ERROR([You need to install python-libxslt1 and python-libxml2]);
-fi
-AC_MSG_RESULT([yes])
-
 ###########################
 # Headless tests
 ###########################
